### ISSUE TITLE
Term `jq1` missing `entry_name`?

### ISSUE THREAD

**Issue by ravwojdyla** [indralab/gilda#98]
Thank you again for all your hard work. We have noticed that one of the grounding terms doesn't appear to have `entry_name`, specifically jq1 famplex. This might be the related row from the grounding terms TSV:

```tsv
jq1     JQ1     CHEBI   CHEBI:95080             curated famplex
```

Is this expected?

**Comment by bgyori** [comment]
The name missing is due to an outdated CHEBI ID being used in the FamPlex resource, and its replacement with a current primary ID hasn't yet propagated into Gilda. I will put out a release later today where this is fixed.

**Comment by bgyori** [comment]
This should be fixed as of 0bd65d7bbe7ad2fe3c43ca103a2c96ae9749fdb6.
